setup.py

```diff
@@ -58,6 +58,7 @@ def run(self):
     cmdclass={"build_py": build_npm_py},
     keywords=["Jupyter"],
     classifiers=["Framework :: Jupyter"],
-    install_requires=["notebook", "nbserverproxy >= 0.8.3"],
+    install_requires=["notebook", "nbserverproxy >= 0.8.6"],
+    dependency_links=['https://github.com/nokome/nbserverproxy/tarball/master#egg=nbserverproxy-0.8.6'],
```

Comment:
Let's remove the dependency_links here. We shouldn't be pointing to your fork in this package, this should refer only to the package version. Plus, I think dependency_links is ignored by default by pip anyway.

Reply:
Agreed. I only added `dependency_links` as a temporary way to test the upstream changes in https://github.com/jupyterhub/nbserverproxy/pull/56. That has been merged now but I suppose we need to wait until a nbserverproxy 0.8.6 is published to PyPI before we can merge this. Or we could just use, `nbserverproxy >= 0.8.5` and merge this now since it doesn't introduce any breaking changes.

Reply:
0.8.5 sounds good, since this is backward-compatible